index.d.ts

```diff
@@ -1601,6 +1639,22 @@ declare module powerbi.extensibility.visual {
         actionId: DialogAction;
         resultState: object;
     }
+
+    export interface ServicePlan {
+        spIdentifier: string;
+        state: ServicePlanState;
+    }
+
+    export interface LicenseInfoResult {
+        /** An array of Service Plans purchased by the active user for this visual */
+        plans: ServicePlan[] | undefined;
+
+        /** Indicates that the visual is being rendered in a Power BI environment that doesn't support licenses management or enforcement. */
+        isLicenseUnsupportedEnv: boolean;
+
+        /** If false, indicates that the licenses info could not be retrieved. */
```

Comment:
hmm this doesn't seem right - how about "Indicates whether the licenses info could be retrieved"